tests/workflows/basic_api_node_with_timeout/test_timeout.py

```diff
@@ -0,0 +1,70 @@
+from unittest.mock import Mock, patch
+
+from vellum.workflows.constants import APIRequestMethod
+from vellum.workflows.nodes.displayable.bases.api_node.node import BaseAPINode
+
+
+class TestAPINodeTimeout:
```

Comment:
[nit] let's move this test over to https://github.com/vellum-ai/vellum-python-sdks/blob/main/src/vellum/workflows/nodes/displayable/api_node/tests/test_api_node.py

`tests/workflows` are intended for full workflows that we'd like to test at the workflow level, calling `workflow.run()` or `workflow.stream()